src/Http/Routing/src/Builder/EndpointRouteBuilderExtensions.cs

```diff
@@ -18,6 +19,8 @@ namespace Microsoft.AspNetCore.Builder;
 /// </summary>
 public static class EndpointRouteBuilderExtensions
 {
+    internal const string MapEndpointTrimmerWarning = "This API may perform reflection on the supplied delegate and its parameters. These types may be trimmed if not directly referenced.";
```

Comment:
Curious: How can a caller invoke these methods w/o referencing passed delegates and other parameters❔

Reply:
The idea is to provide a warning that this may not work. Some patterns just aren’t compatible with trimming